src/pybind/mgr/dashboard/module.py

```diff
@@ -33,7 +33,7 @@
 from .services.sso import SSO_COMMANDS, handle_sso_command
 from .settings import handle_option_command, options_command_list, options_schema_list
 from .tools import NotificationQueue, RequestLoggingTool, TaskManager, \
-    prepare_url_prefix, str_to_bool
+    prepare_url_prefix, str_to_bool, build_url
```

Comment:
`ERROR: /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/module.py Imports are incorrectly sorted and/or formatted.`
```suggestion
    build_url, prepare_url_prefix, str_to_bool
```